### PR TITLE
Chore: Add stackdriver-commiters to owners for terraform-google-cloud-operations repo

### PR DESCRIPTION
**Pull request by hsmatulisgoogle**

### CODE REVIEW

**Review by g-awmalik — APPROVED** [review]

File: infra/terraform/test-org/org/locals.tf [hunk 1]
```diff
@@ -280,6 +280,7 @@ locals {
       description = "Manages Cloud Logging and Cloud Monitoring"
       topics      = local.common_topics.ops
       owners      = ["imrannayer"]
+      groups      = ["stackdriver-committers"]
```
Comment (g-awmalik) [inline infra/terraform/test-org/org/locals.tf:283]: This group doesn't exist in the [TGM](https://github.com/orgs/terraform-google-modules/teams) org where this repo lives.

**Review by g-awmalik — COMMENTED** [review]
@hsmatulisgoogle - please review the comment regarding the `stackdriver-comitters` team.

File: infra/terraform/test-org/org/locals.tf (hunk 1 above)
Comment (bharathkkb) [reply infra/terraform/test-org/org/locals.tf:283]: Hey @g-awmalik, I missed creating this. It should exist now. Feel free to assign to me if you don't have bandwidth to apply.

**Comment by bharathkkb** [comment]
This has been applied

**Pull request merged**